yorkie/src/main/kotlin/dev/yorkie/document/time/TimeTicket.kt

```diff
@@ -9,13 +9,39 @@ internal data class TimeTicket(
     val actorID: ActorID,
 ) : Comparable<TimeTicket> {
 
+    fun toIDString(): String {
+        return "$lamport:$actorID:$delimiter"
+    }
+
+    override fun toString(): String {
```

Comment:
We don't need to override `toString`, `equals` and `hashCode` since `TimeTicket` is a `data class`